README.md

```diff
@@ -1,13 +1,61 @@
 # ini parser
 
-## Try ini parser
+## How to use
 
-```bash
-go run main.go
+- Create a new *.ini file and insert your ini content, for example:
+  
+```ini
+[owner]
+name = John
+organization = threefold
+
+[database]
+version = 12.6
+server = 192.0.2.62
+port = 143
+password = 123456
+protected = true
+```
+
+- Create a new parser struct
+
+```go
+parser := ini.Parser{}
+```
+
+- You can parse a file
+
+```go
+parser.FromFile("*.ini")
+```
+
+- You can parse a string
+
+```go
+str := "[owner]\nname=John\norganization = threefold"
```

Comment:
variable name should be `iniContent` and use backticks for multiline strings instead of squeezing everything in the same str